.github/workflows/k3d.yml

```diff
@@ -22,7 +22,8 @@ jobs:
       - run: sudo apt-get update && sudo apt-get install -y skopeo
       - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633
       - uses: ./actions/setup-tools
-      - run: just sync-k3s-images
+      # The sync may fail when a new K3s version is released. This is expected.
```

Comment:
Should we keep the `continue-on-error: true` above if this suppresses the error from an updated version?

Reply:
Yeah. Since this can fail in general when there are new k3s versions available, we don't want to cancel other workflows just because a new file is generated. It's mostly an advisory check so, when we're doing a new version, for instance, we ensure we've run the sync. This was just a specific weird corner case where the sync would yield an unusable manifest.